js/retentionview.js

```diff
@@ -60,7 +60,8 @@
 					id: model.attributes.id,
 					tagName: _this.tagCollection.get(model.attributes.tagid).attributes.name,
 					timeAmount: model.attributes.timeamount,
-					timeUnit: OCA.File_Retention.RETENTION_UNIT_MAP[model.attributes.timeunit]
+					timeUnit: OCA.File_Retention.RETENTION_UNIT_MAP[model.attributes.timeunit],
+					hasJob: model.attributes.hasJob ? 'yes' : 'no',
```

Comment:
translate?

Reply:
Since nothing was translated, I skipped it here too, but I will try to change that.

Reply:
Lets fix it in a follow up for everything

Reply:
Is there a way to tell the text extractor which values he can expect? Because the following will probably not work:

```
t('files_retention', OCA.File_Retention.RETENTION_UNIT_MAP[model.attributes.timeunit])
```

Reply:
I already commited it, but I can drop it if you like.